library/build.gradle

```diff
@@ -47,7 +51,7 @@ ext {
 
     siteUrl = 'https://github.com/inorichi/tachiyomi-extensions-lib'
     gitUrl = 'https://github.com/inorichi/tachiyomi-extensions-lib.git'
-    githubRepository= 'inorichi/tachiyomi-extensions-lib'
+    githubRepository = 'inorichi/tachiyomi-extensions-lib'
```

Comment:
Not relevant to this PR, but I'll need to eventually remove this stuff since we don't actually publish it to Bintray for use.

Reply:
I'll remove it if you'd like.